core/src/main/java/com/linecorp/armeria/common/FixedHttpResponse.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.stream.FixedStreamMessage;
+
+/**
+ * An {@link HttpResponse} optimized for when all the {@link HttpObject}s that will be published are known at
```

Comment:
An -> A

Reply:
It is 'an' because we read HTTP as eichi ti ti pi - https://www.a-or-an.com/a_an/http

Reply:
oops, I see.